### PR TITLE
Add french translations 🇫🇷

### PR DESCRIPTION
**Pull request by anthony-robin**
This PR adds support for french translations into Lume.

Let me know if additional configuration is needed to make french translations to work :)

### CODE REVIEW

File: packages/utils/src/constants.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ export const FETCH_LIMIT = 20;
 export const LANGUAGES = [
 	{ label: "English", code: "en" },
 	{ label: "Japanese", code: "ja" },
+	{ label: "French", code: "fr" },
```
Comment (anthony-robin) [inline packages/utils/src/constants.ts:6]: Should this label be in french (aka `Français`) or be kept in english ?

**Comment by reyamir** [comment]
Yo, thank for your contribution

**Comment by reyamir** [comment]
Can you help me translate this phase too: "welcome to nostr a social network you control", it isn't present in translate file because I use .png image for it

302470990-1e8e2f31-5309-4634-ab28-666f9146d529

If you have better headline in French, you can suggest it too.

File: packages/utils/src/constants.ts (hunk 1 above)
Comment (anthony-robin) [reply packages/utils/src/constants.ts]: Fixed in commit 👍

**Comment by anthony-robin** [comment]
Your headline phrase could be "Welcome to Nostr, the social network that let you express yourself without any censure !", which would be, in french: "Bienvenue sur Nostr, le réseau social qui vous laisse vous exprimer sans censure !".

Other question: do you plan to use a third party app, such as [Crowdin](https://crowdin.com/), to help dealing with new i18n keys or integrating complete new languages ?

Thank you :)

**Comment by reyamir** [comment]
Hi, I've tested new French headline and it really great 

Screenshot 2024-02-12 at 07 46 06

I've checked Crowdin before, but they need to embed JS package to use translation, I want to avoid using 3rd party API in Lume, only connect to relays and keep IP leak to minimum.

I cannot found other solution to manage translation without embed API, use file to manage seems hard to follow 😅. You can suggest me if you know some service like this

**Comment by anthony-robin** [comment]
Thank you for merging it :)

Why not just using [Crowdin Github Action bot](https://github.com/crowdin/github-action) that will pull/push from/to their service to keep in sync both side (people translating directly from crowdin and people with regular pull requests on the repo) ?

Vitor from [Amethyst](https://github.com/vitorpamplona/amethyst) app has done it. It seems very [light config](https://github.com/search?q=repo%3Avitorpamplona%2Famethyst+crowdin&type=code) on the repo and there is no need to have hard dependency inside the main codebase making it easier to switch to another strategy in the future if needed !

**Comment by reyamir** [comment]
oh, that look good way, I will check it
